README.md

```diff
@@ -0,0 +1,103 @@
+## Introduction
+This is a demo module for Islandora CLAW. It illustrates the modeling of a content type and creation of a drupal module for CLAW.
+
+
+## Steps to create claw_porcus
+### 1 - Create a new Media bundle
+* Go to `/admin/structure/media/add`
+* Provide a label to the bundle: PORCUS_TEXT
+* Leave the type provider as Generic media
+* Go to the Manage fields of the new media bundle: `/admin/structure/media/manage/porcus_text/fields`
+* Add a new file field
+
+### 2 - Create a new Fedora Resource type bundle
+* Go to `/admin/structure/fedora_resource_type`
```

Comment:
Although it probably goes without saying, maybe add something like "and click on the 'Add Fedora Resource Type' button" to remind folks they've got an extra click to go through.